src/service/viewer-impl.js

```diff
@@ -115,21 +115,15 @@ export class Viewer {
     /** @private {number} */
     this.paddingTop_ = 0;
 
+    /** @private {!Object<string, !Observable<!JSONType>>} */
+    this.messageObservables_ = [];
```

Comment:
Actually I want it to be a map of observables, the index is string. Which type should I annotate here?